protobuf/proto/crux.proto

```diff
@@ -629,8 +629,10 @@ message ContainerStatusItem {
   string name = 101;
   string command = 103;
   google.protobuf.Timestamp createdAt = 104;
-  ContainerStatus status = 105; // The 'State' of the container (Created, Running, etc)
-  string state = 106; // The 'Status' of the container ("Created 1min ago", "Exited with code 123", etc)
+  /** The 'State' of the container (Created, Running, etc) */
+  ContainerState state = 105;
```

Comment:
In here we need both? state and status too?

Reply:
Crux might not need it, but core does, so as long as we'd like to support legacy we need both.

Reply:
Okey, we talked about this in person, can we show the developers somehow that we need both right now? I think we can write a small comment with a legacy tag.